outputs.tf

```diff
@@ -2,7 +2,24 @@
 # Outputs
 ########################################################################################################################
 
-#output "myoutput" {
-#  description = "Description of my output"
-#  value       = "value"
-#}
+output "name" {
+  description = "Name of created SCC WP instance."
+  value       = ibm_resource_instance.scc_wp.name
+}
+
+output "id" {
+  description = "ID of created SCC WP instance."
+  value       = ibm_resource_instance.scc_wp.crn
+}
+
+output "sysdig_collector_endpoint" {
+  description = "Sysdig collector endpoint."
+  value       = length(var.scc_wp_keys) > 0 ? ibm_resource_key.scc_wp_keys[keys(var.scc_wp_keys)[0]].credentials["Sysdig Collector Endpoint"] : null
+  sensitive   = true
```

Comment:
this isnt a sensitive value right?

Reply:
output from resource key is `sensitive ibm_resource_key.scc_wp_keys[keys(var.scc_wp_keys)[0]].credentials` and therefore I need to mark that data as sensitive. 

let me know if there is anyway I way I can override it.